pkgs/development/python-modules/joblib/default.nix

```diff
@@ -10,21 +10,27 @@
 
 buildPythonPackage rec {
   pname = "joblib";
-  version = "0.12.4";
+  version = "0.13.2";
 
   # get full repository inorder to run tests
   src = fetchFromGitHub {
     owner = "joblib";
     repo = pname;
     rev = version;
-    sha256 = "06zszgp7wpa4jr554wkk6kkigp4k9n5ad5h08i6w9qih963rlimb";
+    sha256 = "11bspmm2is8akcld2kh6bgp0in1ggdy91qsk5zqybzgapc8b4zdf";
   };
 
+  # python-lz4 compatibility
+  # https://github.com/joblib/joblib/pull/847
+  patches = [ ./lz4_version.patch ];
```

Comment:
please fetch the patch instead

Reply:
Ok, fixed that. However, now building will fail if the pull request ever changes.